Dockerfile

```diff
@@ -255,8 +260,13 @@ RUN sed -i 's|http://archive.ubuntu.com|http://us.archive.ubuntu.com|g' /etc/apt
         sudo -u carma rosdep --rosdistro noetic install --from-paths /home/carma/.base-image/workspace/src --ignore-src -y && \
         sudo -u carma echo "source ~/.base-image/init-env.sh" >> /home/carma/.bashrc && \
         sudo -u carma echo "cd /opt/carma" >> /home/carma/.bashrc && \
+	apt-get update && \
+        apt-get install -y --no-install-recommends ${BASE_DEPS} && \
```

Comment:
Are these two lines needed, since they have already been executed earlier in the Dockerfile?